contracts/voting-verifier/src/contract.rs

```diff
@@ -249,39 +248,6 @@ mod test {
         assert_contract_err_strings_equal(err, ContractError::SourceChainMismatch(source_chain()));
     }
 
-    #[test]
-    fn should_verify_messages_if_not_verified() {
-        let workers = workers(2);
-        let mut deps = setup(workers.clone());
-
-        let msg = ExecuteMsg::VerifyMessages {
-            messages: messages(2),
-        };
-
-        let res = execute(deps.as_mut(), mock_env(), mock_info(SENDER, &[]), msg).unwrap();
-        let reply: VerifyMessagesResponse = from_binary(&res.data.unwrap()).unwrap();
-        assert_eq!(reply.verification_statuses.len(), 2);
-        assert_eq!(
-            reply.verification_statuses,
-            vec![
-                (
-                    CrossChainId {
-                        id: message_id("id", 0),
-                        chain: source_chain()
-                    },
-                    VerificationStatus::None
-                ),
-                (
-                    CrossChainId {
-                        id: message_id("id", 1),
-                        chain: source_chain()
-                    },
-                    VerificationStatus::None
-                ),
-            ]
-        );
-    }
```

Comment:
removed because it was only checking response data and checking other stuff was already considered in other tests

Reply:
Can't we just change this test to call VerifyMessages and then query GetMessageStatuses?

Reply:
this is what I mean with checking other stuff is already done in other tests, changing this test will basically be a duplicate of this one: https://github.com/axelarnetwork/axelar-amplifier/blob/58e98060d86f51e949c1fd7fb4ec2765f5b37e0c/contracts/voting-verifier/src/contract.rs#L529-L562